CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- Fix build on Ubuntu 22.04 when VAAPI support is enabled
```

Comment:
I don't think Ubuntu is worth mentioning; I'd rather say "with old libva" because it likely affects other distributions

Reply:
Done